app/assets/stylesheets/question.css

```diff
@@ -118,7 +118,6 @@ table th a:hover {
   visibility: hidden;
 }
 
-.question-show textarea,
 .question-show pre {
   margin: 0;
```

Comment:
I removed this CSS reference to fix a styling issue in Edit Comment forms. I believe this was done originally to fix #1304, which I think was for the `publiclab.org/questions` route, so unrelated to comments on questions.

This is what the Edit Comment looked like before this change (no border color, no padding):
<img width="692" alt="Screen Shot 2021-01-16 at 9 34 27 PM" src="https://user-images.githubusercontent.com/4361605/104832030-e3ed5600-5842-11eb-9afc-2d7b73313c1e.png">

This is what it looks like after this fix:
<img width="687" alt="Edit Comment After CSS Fix" src="https://user-images.githubusercontent.com/4361605/104832036-ed76be00-5842-11eb-8bef-3d053784f708.png">